CefSharp.Example/Resources/BindingTest.html

```diff
@@ -216,6 +216,19 @@
                 });
             });
 
+            QUnit.test("Async call (receiveArray):", function (assert) {
```

Comment:
Braces on new line, same goes for below

Reply:
Unfortunatelly I have different settings on VS, and have to change all these by hand. :(